examples/README.md

```diff
@@ -172,6 +179,16 @@ Should authentication fail, to prevent automated attacks, a delayed response can
       "failDelay": 5
 ```
 
+#### API keys
+
+Use it to activate API keys
+
+```
+  "http": {
+    "auth": {
+      "apikeys": true
+```
+
```

Comment:
Please also add config file under examples with zot configuration for dex.

Also, we need steps, not sure in what document, on how to configure the dex service itself to work with zot.